### ISSUE TITLE
Incoming Heals Show Up As A Faded Black Bar

### ISSUE THREAD

**Issue by Yujiza** [Aviana/LunaUnitFrames#688]
Hello,

I hope I didn't overlook another person mentioning this already.  I tried to find an answer online before submitting this to no avail.  Thanks for taking a moment to view my issue as I feel it's possibly isolated.

Incoming heals DO show up on player frame for example, however as a faded-background looking bar.  It's not a green bar.  I thought I somehow turned the transparency all the way down to which I didn't.  I changed the color from green to blue to check, nothing changes.  I made sure to download the latest version from the "releases" tab as well.  

![image](https://user-images.githubusercontent.com/53917314/80235040-1e64f200-8627-11ea-93be-fab635276cbc.png)

I thought it was related to saved variables from previous versions, so I got rid of those to start fresh.  No fix.  

The reason I feel it's isolated is because I watch other WoW streamers who use Luna and it shows the bars as green.

Thanks again for having a look.

**Comment by Aviana** [comment]
There are now 3 colors for incoming heals (Your own, others and hots). Do all three not change the color?

**Comment by Yujiza** [comment]
None.  This picture is me bandaging when the bar should be pink:

![image](https://user-images.githubusercontent.com/53917314/80242208-fe3b3000-8632-11ea-9d27-4e69d4e982fd.png)

I also saw this, made sure it was set to "light" although it's for icon use:

![image](https://user-images.githubusercontent.com/53917314/80242251-1448f080-8633-11ea-96b6-f2b79c5bf320.png)

**Comment by Aviana** [comment]
Bandages fall under hots.

**Comment by Yujiza** [comment]
Whoops, my mistake.  Same issue.

![image](https://user-images.githubusercontent.com/53917314/80243548-6be85b80-8635-11ea-9dc4-a4ee0be356cc.png)

**Comment by Aviana** [comment]
From the screenshots you posted i am pretty sure that your UI mod is replacing the blizzard texture "UI-Tooltip-Background". Thats why its black.

**Comment by Yujiza** [comment]
Aviana, thank you so very much!

In this scenario, it wasn't an addon that was effecting it, but rather a UI skin set.  I was able to pinpoint exactly what it was as you said.  I'll know for future reference.

The reason I didn't come to the conclusion sooner is because other unitframe addons didn't show this issue.  They were all green.  I highly prefer LunaUnitFrames over all else, and when none of them could satisfy my thirst, I figured I take a tab at this one last time.
